### ISSUE TITLE
Icon missing for metacritic show overview and movies

### ISSUE THREAD

**Issue by theotherp** [trossr32/sonarr-radarr-lidarr-autosearch-browser-extension#184]
**Describe the bug**
Icon missing for metacritic show overview and movies

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://www.metacritic.com/tv/the-jinx-the-life-and-deaths-of-robert-durst/ or https://www.metacritic.com/movie/civil-war/
2. Find no icon

The container selector `div[class*="productHero_title"] > div` works for https://www.metacritic.com/tv/the-jinx-the-life-and-deaths-of-robert-durst/season-2/:
```
<div class="c-productHero_seasonsComponent g-outer-spacing-top-large">
   <div class="c-productHero_seasonsComponent_season">
      Season 2
   </div>
   <div class="c-seasonsComponent g-outer-spacing-top-small"></div>
</div>
```
but not for https://www.metacritic.com/tv/the-jinx-the-life-and-deaths-of-robert-durst/ because here the title looks like this:

```
<div class="c-productHero_title g-inner-spacing-bottom-medium g-outer-spacing-top-medium u-grid">
   <h1>The Jinx: The Life and Deaths of Robert Durst</h1>
</div>
```

Same for movies https://www.metacritic.com/movie/civil-war/:
```
<div class="c-productHero_title g-inner-spacing-bottom-medium g-outer-spacing-top-medium">
   <h1>Civil War</h1>
</div>
```

Thanks!

**Comment by trossr32** [comment]
Caused by metacritic site redesign. Will be fixed in v2.8.7

**Comment by theotherp** [comment]
Great, confirmed working. Thanks!
